lib/google/ads/google_ads/google_ads_client.rb

```diff
@@ -225,7 +225,6 @@ def get_updater_proc
         # Provides a Google::Auth::Credentials initialized with a keyfile
         # specified in the config.
         def get_service_account_credentials
-          raise 'config.impersonate required if keyfile specified' unless @config.impersonate
```

Comment:
Could you add a test to verify this works as expected?

https://github.com/googleads/google-ads-ruby/blob/main/test/test_google_ads_client.rb

Reply:
Updated the test. It still will raise for an OpenSSL error but that means its actually reading the test file so the `keyfile` config is valid.